pkg/testutils/testutils.go

```diff
@@ -540,3 +541,15 @@ func AssertNodeSpecificCertificates(t *testing.T, node *corev1.Node, secrets []c
 
 	require.Equalf(t, 0, expectedSet.Len(), "missing certificates for node: %v", expectedSet.List())
 }
+
+func CreateEtcdBackupSpec(timezone, schedule string) backupv1alpha1.EtcdBackupSpec {
```

Comment:
if it's still unused, just return a pointer to the spec directly, no need for the other function

Reply:
done 👍🏽